### DISCUSSION TITLE
Build failed for 2.6.6 on macOS 13.0

### DISCUSSION THREAD

**Discussion by syncopated** [rbenv/ruby-build#2080] [Build failures] [answered]
Hey All 👋,

Wondering if someone might be able to help me figure how install v2.6.6 on macOS 13.0. I'm on an Intel Mac, have openssl1.1 installed via Homebrew and passed `--with-openssl-dir` to ruby-build via `RUBY_CONFIGURE_OPTS`

Anything stand out in the [build Log](https://gist.github.com/syncopated/823283078fe01f2b18da892bbca1ff15)?

**Comment by mislav** [comment, marked as answer]
Newest Xcode Command Line Tools is known to give people problems. I've received the same error as you (although on an M1 machine) and the solution was to either pass `--enable-shared` (although, it should be enabled by default) via RUBY_CONFIGURE_OPTS or to downgrade Xcode Command Line Tools https://bugs.ruby-lang.org/issues/18912

Note that your macOS version is very new and that the Ruby version you're trying to compile is very old and unsupported (EOL). You should try to migrate your project to Ruby 3.0 or newer.

**Reply by syncopated** [reply]
Thanks. I tried passing `--enable-shared` and that didn't work (it *is* enabled by default). Will see if I can downgrade Xcode.
